config/localmodelnodes/manager.yaml

```diff
@@ -25,23 +25,24 @@ spec:
         kserve/localmodel: worker
       serviceAccountName: kserve-localmodelnode-agent
       securityContext:
-        runAsNonRoot: true
-        seccompProfile:
-          type: RuntimeDefault
+        runAsUser: 0 # TODO: remove this to run as non-root https://issues.redhat.com/browse/RHOAIENG-33247
```

Comment:
If the target maturity is Dev Preview, it is OK. But if target is TP or GA, I don't see this good.